Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.18-alpine
+
+WORKDIR /app
+
+COPY go.mod go.sum ./
+
+RUN go mod download
+
+COPY *.go ./
+COPY tscanner ./tscanner
+
+RUN CGO_ENABLED=0 GOOS=linux go build -o Terrapin-Scanner
+
+ENTRYPOINT [ "/app/Terrapin-Scanner" ]
```

Comment:
Like this:

```suggestion
FROM golang:1.18-alpine AS builder

WORKDIR /app

COPY go.mod go.sum ./

RUN go mod download

COPY *.go ./
COPY tscanner ./tscanner

RUN CGO_ENABLED=0 GOOS=linux go build -o terrapin-scanner


FROM gcr.io/distroless/base-debian12 AS build-release-stage

WORKDIR /

COPY --from=builder /app/terrapin-scanner /terrapin-scanner

USER nonroot:nonroot

CMD [ "/terrapin-scanner" ]
```

Reply:
Debian is actually a heavier image than Go. I decided to use the code from https://github.com/RUB-NDS/Terrapin-Scanner/pull/19#discussion_r1444678169.